packages/form-js-viewer/README.md

```diff
@@ -154,9 +154,33 @@ Remove form from the document.
 
 ## Events
 
-The form emits the `changed` and `submit` events you may hook into.
+### `changed :: { data, errors }`
+Fired off every time there is a form state change.
 
-Both events receive `{ data, errors }` as a payload.
+### `submit :: { data, errors }`
+Fired off on form submission.
+
+### `import.done :: { error, warnings }`
+Fired whenever a schema has finished importing, whether it succeeds or fails.
+
+### `form.layoutCalculated :: { rows }`
+
+### `form.layoutCleared :: {}`
+
+### `<lifeCycleEvent> :: {}`
+- `detach`
+- `attach`
+- `form.init`
+- `form.clear`
+- `form.destroy`
+- `diagram.clear`
+- `diagram.destroy`
+
+### `<formFieldEvent> :: { formField }`
+- `formField.add` 
+- `formField.remove` 
```

Comment:
These are editor events, correct? Maybe we can add the full event documentation to the general `form-js` README?

Reply:
Actually no these events are bound to the form field registry. So they technically are viewer events, they would fire during import.

Reply:
I think it's still good to separate them, as you access them differently. form.on and formEditor.on.

Reply:
You can programmatically add fields to the viewer in theory, there may be other use cases we haven't thought of that do this. For example, a streamed version of the form generator using only a viewer.

Reply:
Good catch!